### ISSUE TITLE
Count Statistics file not generated

### ISSUE THREAD

**Issue by StarkTS98** [KindLab/scDamAndTools#1]
Hello, I am new to the scDamAndTools. I am following the tutorial given on the repository. So, I am using the `generate_celseq_counts.py` for generating UMI-unique transcript counts. I have my reference genome index files, GTF file and the aligned .bam files which creates a count file correctly. I also want to save the count statistics and the `--save-stats` argument allows one to do that. However, after running the script, it does not generate a stats tsv file for me. I have checked, my version of scDamAndTools does support the `--save-stats` argument. I tried giving a file name for saving the stats but it doesn't allow that. 
Any help would be appreciated.

**Comment by krooijers** [comment]
Indeed, this is reproducible, but only when `--outfile` is not given.

Without `--outfile` a table is printed to STDOUT.
See: <https://github.com/KindLab/scDamAndTools/blob/3c907402338295f6e6809c82ac804b755a167746/scripts/generate_celseq_counts.py#L228>
